benchmarks/bm_runner.py

```diff
@@ -85,9 +86,8 @@ def _prep_data_gen_env() -> None:
         )
         # Find the environment built above, set it to be the data generation
         #  environment.
-        data_gen_python = next(
-            (ROOT_DIR / ".nox").rglob(f"tests*/bin/python{python_version}")
-        ).resolve()
```

Comment:
There's a bit here I don't understand...
Previously the end part of this path construction had the equivalent of .. 
`data_gen_python = (env_directory / "bin" / "python{python_version}").resolve()`
But now we have lost the `{python_version}` part of this.
Does that still work + if so why ??

Reply:
It was unnecessary, since all environments include various symlinks in their `bin/` directory (using Python 3.13 as the example):

- `python` -> `python3.13`
- `python3` -> `python3.13`
- `python3.1` -> `python3.13`
- `python3.13`

Removing this unnecessary element hopefully makes the file less complex in future templating.